BitmarkRegistry/Extensions/String+Extension.swift

```diff
@@ -10,6 +10,10 @@ import Foundation
 
 extension String {
   func middleShorten(eachMaxChars: Int = 4) -> String {
+    if let currentAccount = Global.currentAccount, currentAccount.getAccountNumber() == self {
```

Comment:
It violates single responsibility principle.
I'd suggest to separate into another function, like in an extension of `AccountNumber`.